lib/java/com/google/android/material/slider/BaseSlider.java

```diff
@@ -1206,6 +1212,11 @@ public void setTrackInactiveTintList(@NonNull ColorStateList trackColor) {
     invalidate();
   }
 
+  /**
+   * Enable or disable the Slider
+   * @see #isEnabled()
+   * @attr ref com.google.android.material.R.styleable#Slider_android_enabled
+   */
```

Comment:
this comment shouldn't be needed, as by default the doc from the parent method is inherited

Reply:
You are right 👍